helm/kiam/README.md

```diff
@@ -27,6 +27,35 @@ $ helm install uswitch/kiam --name my-release
 
 The command deploys kiam on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
+## Key Configuration
+The default helm configuration will probably not work out-of-the-box. You will most likely need to adjust the following:
+
+* Kiam requires SSL certs from the host system to be mounted into the kiam-server pod in order to be able to contact the AWS meta-data API. If the SSL cert directory on the host(s) you intend to run the kiam-server on does not match the default (`/usr/share/ca-certificates`), you will need to set the `server.sslCertHostPath` variable.
```

Comment:
Clarify that this is adding globally trusted root certificates, not kiam-specific certificates.
```suggestion
* Kiam requires trusted SSL root certificates from the host system to be mounted into the kiam-server pod in order to be able to contact the AWS meta-data API. If the SSL cert directory on the host(s) you intend to run the kiam-server on does not match the default (`/usr/share/ca-certificates`), you will need to set the `server.sslCertHostPath` variable.
```

Reply:
Good spot, thanks!